expression/collation.go

```diff
@@ -122,6 +122,18 @@ var (
 		charset.CollationUTF8MB4: 5,
 		charset.CollationBin:     6,
 	}
+
+	// CollationStrictness indicates the strictness of comparison of the collation. The partial order in a weak collation also holds in a strict collation.
+	// For example, if a < b in a weak collation(e.g. general_ci), then there must be a < b in a strict collation(e.g. _bin).
```

Comment:
Is this "Strictness" rule exists in MySQL?

Reply:
No, but I think it's ture.